hack/verify-govulncheck.sh

```diff
@@ -36,7 +36,8 @@ kube::util::ensure-temp-dir
 WORKTREE="${KUBE_TEMP}/worktree"
 
 # Create a copy of the repo with $BRANCH checked out
-git worktree add -f "${WORKTREE}" "${BRANCH}"
+git worktree add -f "${WORKTREE}" "${BRANCH}" -q
+
```

Comment:
Please put the -q directly after the add, so it will be easier to read and reduce the chance of mistakes when this is modified in the future

Reply:
Put it after -f (cause of alphabet order and -f option is not relates to branch names

Reply:
Following the documentation for the [git worktree](https://git-scm.com/docs/git-worktree) I would agree, that the order shall be alphabetically.

Maybe a stylish-"enhancement" (or a personal flavor decision?) would be to combine those to arguments into a single argument. Like "-fq". 

WDYT? 🙂

Reply:
fine by me :-D